src/main.mjs

```diff
@@ -62,6 +71,14 @@ export default {
       category: "css-declaration-sorter",
       default: true,
     },
+    propertiesOrder: {
+      type: "string",
+      array: true,
+      description:
+        "An array of declaration names to sort according to their index in the array.",
+      category: "css-declaration-sorter",
+      default: [{ value: [] }],
+    },
```

Comment:
Could passing an array be one of the choices in `cssDeclarationSorterOrder`? Or is that invalid/unsupported by Prettier?

Reply:
As i can see here support any type:
https://github.com/prettier/prettier/blob/d970deed6b6cabe21fdb35249265d766932b5718/src/index.d.ts#L753-L762
But at the moment of using the normalizer inside prettier shows that the parameters are invalid. Maybe I'm missing something, but that's what I see in the source code.

Reply:
I see it now, sadly Prettier doesn't support arbitrary values as a choice, ah well. :)